packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/FloatingLink/InsertLinkModal/InsertModal.tsx

```diff
@@ -54,32 +53,15 @@ export const InsertModal: FC<Props> = ({
         </FormControl>
 
         {children}
-
-        <div className="tw-pt-5">
-            <FormControl
-                label={{
-                    children: 'URL',
-                    htmlFor: 'url',
-                    required: true,
-                }}
-            >
-                <TextInput
-                    id="url"
-                    value={state.url}
-                    placeholder="https://example.com"
-                    focusOnMount
-                    onChange={onUrlChange}
-                />
-            </FormControl>
-            {!isValidUrlOrEmpty(state?.url) && <div className="tw-text-red-65 tw-mt-3">Please enter a valid URL.</div>}
-        </div>
-
-        <div className="tw-mt-3">
-            <LinkSelector url={state.url} appBridge={appBridge} onUrlChange={onUrlChange} />
-        </div>
-
-        <div className="tw-mt-3">
-            <Checkbox value="new-tab" label="Open in new tab" state={state.newTab} onChange={onToggleTab} />
+        <div className="tw-mt-5">
+            <LinkComponent
```

Comment:
Maybe LinkInput makes more sense in terms of naming? What do you think?

Reply:
Also, can we import `LinkComponent` from brand sdk so it does not need to be here?

Reply:
Agree. renamed it.

Yes, that would be the plan. But it's not yet moved. Is it?